prow/cmd/deck/static/prow/prow.ts

```diff
@@ -652,16 +653,22 @@ function redraw(fz: FuzzySearch): void {
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
     const url = `https://${window.location.hostname}/rerun?prowjob=${prowjob}`;
     const c = document.createElement("td");
-    const i = icon.create("refresh", "Show instructions for rerunning this job");
-    i.onclick = () => {
-        modal.style.display = "block";
-        rerunElement.innerHTML = `kubectl create -f "<a href="${url}">${url}</a>"`;
-        const copyButton = document.createElement('a');
-        copyButton.className = "mdl-button mdl-js-button mdl-button--icon";
-        copyButton.onclick = () => copyToClipboardWithToast(`kubectl create -f "${url}"`);
-        copyButton.innerHTML = "<i class='material-icons state triggered' style='color: gray'>file_copy</i>";
-        rerunElement.appendChild(copyButton);
-    };
+    var i;
+    if (rerunCreatesJob) {
+      i = icon.create("refresh", "Rerun this job");
+      i.href = `${url}`;
```

Comment:
would like to see a pop up like `created prowjob $prowjobID`, or even a confirm page can help?

Reply:
Yeah, had that thought as well. Will ask one of our frontend devs for help on Monday.